keyboards/newgame40/readme.md

```diff
@@ -0,0 +1,17 @@
+# NEWGAME40
+
+![NEWGAME40](https://i.imgur.com/HtujXda.jpg)
+
+40% (4x10) ortholinear keyboard.
+
+Keyboard Maintainer:
+[GoTakigawa](https://github.com/GoTakigawa)
+[@Go_Drums](https://twitter.com/Go_Drums)
```

Comment:
This line should have two spaces at the end so GitHub presents the text as intended.

```suggestion
[@Go_Drums](https://twitter.com/Go_Drums)  
```